src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -27,36 +28,39 @@ protected override TestWorkspace CreateWorkspace(string fileContents)
             return new CSharpCompletionService(workspace, exclusiveProviders);
         }
 
-        protected override Task BaseVerifyWorkerAsync(
+        private protected override Task BaseVerifyWorkerAsync(
             string code, int position,
             string expectedItemOrNull, string expectedDescriptionOrNull,
             SourceCodeKind sourceCodeKind, bool usePreviousCharAsTrigger, bool checkForAbsence,
-            int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix)
+            int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix,
+            List<CompletionItemFilter> matchingFilters, bool targetTypedExperimentEnabled)
```

Comment:
>List [](start = 12, length = 4)

Why List?

Reply:
No reason. It could be IEnumerable. If you have a strong reason for changing it, then can we do it in the follow-up pass to clean up this list anyway?

Reply:
OK. Let us do this in a follow up: [https://github.com/dotnet/roslyn/issues/34995](https://github.com/dotnet/roslyn/issues/34995)

---
In reply to: [275565362](https://github.com/dotnet/roslyn/pull/34655#discussion_r275565362) [](ancestors = 275565362)